pulsar-client/src/main/java/org/apache/pulsar/client/impl/ProducerBuilderImpl.java

```diff
@@ -223,6 +224,13 @@ private ProducerBuilderImpl(PulsarClientImpl client, ProducerConfigurationData c
         return this;
     }
 
+    @Override
+    public ProducerBuilder<T> batchingContainerBuilder(BatchMessageContainerBuilder batchingContainerBuilder) {
```

Comment:
I am not sure if `batchingContainerBuilder` is the best name. maybe `MessageBatcher`?

Reply:
I've also struggled with this problem, if we named messageBatcher, It looks like it's provided with a message batcher, but in fact, it provides the ability to build a message batcher. because if it is a partitioned topic, each partitions producer can not use same message batcher, so i named batchingContainerBuilder. This naming seems a bit complicated, how a bout `batcherBuilder` ?

Reply:
Batcher sounds like a better name.